library-best-practices/JLBP-8.md

```diff
@@ -0,0 +1,14 @@
+[JLBP-8] Advance widely used functionality to a stable version
+--------------------------------------------------------------
+
+Libraries at 0.x versions and unstable features marked with annotations make no
```

Comment:
"unstable features marked with annotations" might not be clear out of context. Readers might ask "Isn't is true that _all_ unstable features make no promises about stability, not just those with annotations?"

Maybe something like: "Some libraries expose APIs but make no promises of stability. This includes all APIs of libraries at 0.x versions, and it may include specially annotated APIs in an otherwise stable library."

Reply:
I will make this statement clearer.